src/test/regress/expected/local_shard_execution.out

```diff
@@ -1571,21 +1571,16 @@ CALL register_for_event(19, 1, 'yes');
 SET citus.log_local_commands TO ON;
 SET client_min_messages TO DEBUG2;
 CALL register_for_event(19, 1, 'yes');
-NOTICE:  executing the command locally: INSERT INTO public.event_responses_1220112 AS citus_table_alias (event_id, user_id, response) VALUES (19, 1, 'yes'::public.invite_resp) ON CONFLICT(event_id, user_id) DO UPDATE SET response = excluded.response
-DEBUG:  Distributed planning for a fast-path router query
-DEBUG:  Creating router plan
-DEBUG:  Plan is router executable
-NOTICE:  executing the command locally: SELECT count(*) AS count FROM public.event_responses_1220112 event_responses WHERE (event_id OPERATOR(pg_catalog.=) 19)
-DEBUG:  Distributed planning for a fast-path router query
-DEBUG:  Creating router plan
-DEBUG:  Plan is router executable
-NOTICE:  executing the command locally: UPDATE public.event_responses_1220112 event_responses SET response = $3 WHERE (event_id OPERATOR(pg_catalog.=) 19)
+NOTICE:  executing the command locally: INSERT INTO public.event_responses_1330001 AS citus_table_alias (event_id, user_id, response) VALUES (19, 1, 'yes'::public.invite_resp) ON CONFLICT(event_id, user_id) DO UPDATE SET response = excluded.response
+NOTICE:  executing the command locally: SELECT count(*) AS count FROM public.event_responses_1330001 event_responses WHERE (event_id OPERATOR(pg_catalog.=) 19)
+NOTICE:  executing the command locally: SELECT count(*) AS count FROM (SELECT NULL::integer AS event_id, NULL::integer AS user_id, NULL::public.invite_resp AS response WHERE false) event_responses(event_id, user_id, response) WHERE ((event_id OPERATOR(pg_catalog.=) 19) AND false)
+NOTICE:  executing the command locally: UPDATE public.event_responses_1330001 event_responses SET response = 'yes'::public.invite_resp WHERE (event_id OPERATOR(pg_catalog.=) 19)
 -- should be fine even if no parameters exists in the query
 SELECT count(*) FROM event_responses WHERE event_id = 16;
 DEBUG:  Distributed planning for a fast-path router query
 DEBUG:  Creating router plan
 DEBUG:  Plan is router executable
-NOTICE:  executing the command locally: SELECT count(*) AS count FROM public.event_responses_1220112 event_responses WHERE (event_id OPERATOR(pg_catalog.=) 16)
```

Comment:
Let's put `SET citus.next_shard_id TO 1480000;` after reconnecting to master so this shard number is stable

Reply:
done